pkgs/development/python-modules/ephemeral-port-reserve/default.nix

```diff
@@ -0,0 +1,16 @@
+{ stdenv, buildPythonPackage, fetchPypi }:
+buildPythonPackage rec {
```

Comment:
I believe this is an application, not a library. In this case, it must use `buildPythonApplication` and be moved out of `pythonPackages`.
This file should be moved somewhere under `pkgs/tools/networking`.